geoapi/services/features.py

```diff
@@ -240,15 +239,22 @@ def fromShapefile(database_session, projectId: int, fileObj: IO, metadata: Dict,
         return features
 
     @staticmethod
-    def fromRAPP(database_session, projectId: int, fileObj: IO, metadata: Dict, original_path: str = None) -> Feature:
+    def from_rapp_questionnaire(database_session, projectId: int, fileObj: IO,
+                                additional_files: List[IO], original_path: str = None) -> Feature:
         """
+        Import RAPP questionnaire
+
+        RAPP questionnaire is imported along with any asset images that it
+        refers to. The asset images are assumed to reside in the same directory
+        as the questionnaire .rq file.
 
         :param projectId: int
-        :param fileObj: file descriptor
-        :param metadata: Dict of <key, val> pairs
+        :param fileObj: questionnaire file
```

Comment:
Where is the `questionnaire_file` value coming from? Its probably just me, but I do not see this defined anywhere before it is used here.

Reply:
Maybe if it specified `questionnaire rq file` it would be clearer?